### ISSUE TITLE
BatchNormalization make bad results

### ISSUE THREAD

**Issue by kushia** [keras-team/keras#8542]
Hi everyone,
I tried to add somes BatchNormalization layers into my network today.
Without BatchNormalization, I get a decent 75% of accuracy on the test set.
But when I add BatchNormalization layers, my train accuracy increase, but my test accuracy remains to 50. (Or increase a little bit, to 55)
What could be the cause ? I use BatchNorm with default values (Since I use tf ordering, that's the good thing to do I guess)
For conv1d and dense layers.
I tried to add them after Activation Layer, and before it, but the results stay the same.. Do you have an idea of why it's stuck, or had a similar problem ?

**Comment by jeremydr2** [comment]
Do you have any dropout layers in your network before BatchNorm? That can cause train accuracy to appear high but test accuracy to stay low.

This is really a stackoverflow question, not a github issue.
